cmd/clef/tutorial.md

```diff
@@ -31,43 +31,51 @@ NOTE: This file does not contain your accounts. Those need to be backed up separ
 
 ## Creating rules
 
-Now, you can create a rule-file.
+Now, you can create a rule-file. Note that it is not mandatory to use predefined rules, but it's good practice.
```

Comment:
Well, I wouldn't say good practice, it's more a usability thing. Without any rules, you'll have to manually approve every single operation.